src/Exporter.php

```diff
@@ -236,7 +236,7 @@ protected function recursiveExport(&$value, $indentation, $processed = null)
             }
 
             return "'" .
-            str_replace(array("\r\n", "\n\r", "\r"), array("\n", "\n", "\n"), $value) .
+            str_replace(array("\r\n", "\n\r", "\r", "\n"), array('<crlf>', '<lfcr>', '<lf>', '<ln>'), $value) .
```

Comment:
You also need to make sure that this only gets applied to a string. Look at the test failures in https://travis-ci.org/sebastianbergmann/exporter/jobs/217512272 to see what I mean.